reference/java/README.md

```diff
@@ -1 +1,66 @@
-# LAS Java SDK
+# Java SDK for Lucidtech AI Services API
+
+## Usage
+
+### Preconditions
+
+- Documents must be in upright position
+- Only one receipt or invoice per document is supported
+- Supported file formats are: jpeg, pdf
+
+### Quick start
+
+```java
+import ai.lucidtech.las.sdk.Client;
+import ai.lucidtech.las.sdk.Prediction;
+import ai.lucidtech.las.sdk.Field;
+
+import java.util.List;
+
+public class Main {
+    public static void main(String[] args) throws IOException, URISyntaxException {
+        Credentials credentials = new Credentials(
+            clientId,
+            clientSecret,
+            apiKey,
+            authEndpoint,
+            apiEndpoint
+        );
+        Client client = new Client("<api endpoint>");
+        JSONObject document = this.createDocument(documentContent, contentType, consentId);
+        String documentId = document.getString("documentId");
+        JSONObject prediction = this.createPrediction(documentId, modelName);
+
+        fields.forEach(item -> {
+            JSONObject field = (JSONObject) item;
+            System.out.println("field: " + field.getString("label"));
+            System.out.println("field: " + field.getString("value"));
+            System.out.println("field: " + field.getFloat("confidence"));
+        });
+    }
+}
+```
```

Comment:
Can you add examples for the other endpoints as well? We want the same quickstart examples as JS has. Maybe we could also drop the class definition in the quickstart examples and only show the necessary code.